scrapers/scraper.py

```diff
@@ -0,0 +1,434 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""Usage: python <name>.py [-s] [-u]
+   Output: Outputs data (necessary for the family visualizer) for languages in json files
+   To change languages, change the languages, iso3to2 and family variables
+"""
+
+from collections import Counter
+from pathlib import Path
+import xml.etree.ElementTree as etree
+import argparse
+import re
+import sys
+import logging
+import os
+import json
+import requests
+import subprocess
+import shutil
+import stat
+
+from lexccounter import countStems as countLexcStems
+from dixcounter import get_info as countDixStems
+
+languages = [
+    "aze",
+    "bak",
+    "chv",
+    "crh",
+    "kaa",
+    "kaz",
+    "kir",
+    "kjh",
+    "kum",
+    "nog",
+    "sah",
+    "tat",
+    "tuk",
+    "tur",
+    "tyv",
+    "uig",
+    "uzb",
+]
+iso3to2 = {
+    "aze": "az",
+    "bak": "ba",
+    "chv": "cv",
+    "crh": "crh",
+    "kaa": "kaa",
+    "kaz": "kk",
+    "kir": "ky",
+    "kjh": "kjh",
+    "kum": "kum",
+    "nog": "nog",
+    "sah": "sah",
+    "tat": "tt",
+    "tuk": "tk",
+    "tyv": "tyv",
+    "tur": "tr",
+    "uig": "ug",
+    "uzb": "uz",
+}
+pairLocations = ["incubator", "nursery", "staging", "trunk"]
+langLocations = ["languages", "incubator"]
+states = ["prototype", "development", "working", "production"]
+family = "Turkic"
+wikiURL = "http://wiki.apertium.org/wiki/" + family + "_languages"
+
+
+def del_rw(action, name, exc):
+    """Force deletes file if access is denied"""
+    os.chmod(name, stat.S_IWRITE)
+    os.remove(name)
+
+
+def monoHistory(language, path):
```

Comment:
If `path` here is never going to change, would it make more sense as a constant at the top of the file?